go/dpagg/count.go

```diff
@@ -184,16 +184,18 @@ func (c *Count) ThresholdedResult(thresholdDelta float64) *int64 {
 	return &result
 }
 
-// ComputeConfidenceInterval computes a confidence interval with integer bounds that contains the true count with
-// a probability greater or equal to 1 - alpha using the noised count computed by Result().
+// ComputeConfidenceInterval computes a confidence interval with integer bounds that contains the true count
+// with a probability greater than or equal to 1 - alpha using the noised count computed by Result().
+// Note Result() needs to be called before ComputeConfidenceInterval, otherwise this will return an error.
```

Comment:
Could you please put an empty line before `Note...`? This would make it a separate paragraph in https://godoc.org/github.com/google/differential-privacy/go/dpagg